conda/recipes/cugraph/meta.yaml

```
@@ -25,13 +25,16 @@ build:
 
 requirements:
   build:
+    - cmake {{ cmake_version }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }} {{ cuda_version }}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - python x.x
     - cython>=0.29,<0.30
+    - cmake>=3.20.1,!=3.23.0
```

I'm not sure if this is necessary. Could you try removing it and seeing? I don't think we need it in the host environment.

In case we do need it, though, it should use the same `cmake_version`. I'm guessing that this is copied from the cudf PR; if so I can make the fix in cudf once you've confirmed whether or not this requirement is actually necessary. Thanks!
```suggestion
    - cmake {{ cmake_version }}
```

---

There is a bug in 3.23.0 where the Ninja generator doesn't work at all when the user has requested a clang compile database to be built.
Since things like rapids-compose request the generation of the compile database, we need to exclude 3.23.0 as I fixed the issue for 3.23.1

---

Yes, but that is [also encoded in the `cmake_version`](https://github.com/rapidsai/cugraph/pull/2378/files#diff-be44d5b908c4df0f4677afe316a6b2521736f580ed2e93a79cbb4562603ac304R11). I just want to use the same version in both the `build` and `host` sections (assuming that it's even necessary.

---

Ahhh okay, that makes sense.